code/game/machinery/ecto_sniffer.dm

```diff
@@ -34,11 +42,15 @@
 /obj/machinery/ecto_sniffer/proc/activate(mob/activator)
 	flick("ecto_sniffer_flick", src)
 	playsound(loc, 'sound/machines/ectoscope_beep.ogg', 25)
-	visible_message("<span class='notice'>[src] beeps, detecting ectoplasm! There may be additional positronic brain matrixes available!</span>")
+	//NSV13 - Ecto Sniffer Radio Yelling - Start
+	var/msg = "[src] beeps, detecting ectoplasm! There may be additional positronic brain matrices available!"
+	radio.talk_into(src, msg, RADIO_CHANNEL_SCIENCE)
+	//NSV13 - Ecto Sniffer Radio Yelling - Stop
+	visible_message("<span class='notice'></span>")
```

Comment:
Is this an empty visible message?

Reply:
Yes, it appears that I forgot to remove it

Reply:
I like the idea that it just posts an empty line in everyone's chat bars